src/standalone/Soil/soil_albedo.jl

```diff
@@ -0,0 +1,200 @@
+export update_albedo!, CLMTwoBandSoilAlbedo, ConstantTwoBandSoilAlbedo
+
+abstract type AbstractSoilAlbedoParameterization end
+
+"""
+    ConstantTwoBandSoilAlbedo{
+        SF <: Union{AbstractFloat, ClimaCore.Fields.Field},
+    } <: AbstractSoilAlbedoParameterization
+
+A parameterization for soil albedo: the soil albedo is 
+defined in two bands (PAR and NIR), can spatially vary or
+be set to scalar, but is assumed not to vary in time (or
+with water content).
+"""
+struct ConstantTwoBandSoilAlbedo{
+    SF <: Union{AbstractFloat, ClimaCore.Fields.Field},
+} <: AbstractSoilAlbedoParameterization
+    "Soil PAR Albedo"
+    PAR_albedo::SF
+    "Soil NIR Albedo"
+    NIR_albedo::SF
+end
+
+"""
+    update_albedo!(bc::AtmosDrivenFluxBC, albedo::ConstantTwoBandSoilAlbedo, p, soil_domain, model_parameters)
+
+Updates PAR and NIR albedo using the constant parameters provided in `albedo`.
+"""
+function update_albedo!(
+    bc::AtmosDrivenFluxBC,
+    albedo::ConstantTwoBandSoilAlbedo,
+    p,
+    soil_domain,
+    model_parameters,
+)
+    @. p.soil.PAR_albedo = albedo.PAR_albedo
+    @. p.soil.NIR_albedo = albedo.NIR_albedo
+end
+
+"""
+     CLMTwoBandSoilAlbedo{
+        FT <: AbstractFloat,
+        SF <: Union{FT, ClimaCore.Fields.Field},
+    } <: AbstractSoilAlbedoParameterization
+
+A parameterization for soil albedo: the soil albedo is 
+defined in two bands (PAR and NIR), can spatially vary or
+be set to scalar, and varies with water content at the surface
+θ_sfc, according to CLM:
```

Comment:
If the soil albedo is set to a scalar, is it still adjusted based on the surface water content, or would it be independent?